src/overlay/test/FloodTests.cpp

```diff
@@ -154,7 +155,8 @@ TEST_CASE("Flooding", "[flood][overlay][acceptance]")
             auto msg = tx1->toStellarMessage();
             auto res = inApp->getHerder().recvTransaction(tx1, false);
             REQUIRE(res == TransactionQueue::AddResult::ADD_STATUS_PENDING);
-            inApp->getOverlayManager().broadcastMessage(msg);
+            inApp->getOverlayManager().broadcastMessage(msg, false,
```

Comment:
grep for `broadcastMessage` - there are others that need updated (not sure why those pass for you, with pull mode enabled they should crash as there is no hash passed in), see `OverlayManagerTest.cpp`

Reply:
Fixe.d